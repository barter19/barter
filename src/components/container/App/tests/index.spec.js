import { shallowMount } from '@vue/test-utils';
import AppComponent from '../index';

const renderer = require('vue-server-renderer').createRenderer();

describe('App Component', () => {
  const ClonedComponent = shallowMount(AppComponent);
  const { vm } = ClonedComponent;

  it('renders to a nice snapshot', () => {
    renderer.renderToString(vm, (err, str) => {
      expect(str).toMatchSnapshot();
    });
  });
});
