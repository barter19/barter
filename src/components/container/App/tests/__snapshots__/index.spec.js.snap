// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders to a nice snapshot 1`] = `
<div id="app">
  <Header-stub></Header-stub>
  <router-view></router-view>
  <Footer-stub></Footer-stub>
</div>
`;
